src/test/java/command/DeadlineCommandTest.java

```diff
@@ -0,0 +1,26 @@
+package command;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import java.time.LocalDate;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import storage.TaskList;
+import task.Deadline;
+import task.Task;
+
+public class DeadlineCommandTest {
```

Comment:
test can follow this `featureUnderTest_testScenario_expectedBehavior()` according to guideline.